chore: update sidebars (#156)
c6o/docs: content/_404.md

# This page does not exist

### Here's what might have happened

- You may have mistyped the URL
- The page was moved
- The page never existed

> Please reach out to us by filling an issue on our [Github Repository](https://github.com/c6o/docs/issues/new?assignees=&labels=bug&template=bug-report.md&title=) or by letting us know in [Discord](https://discord.gg/wx3JkVjTPy)

Reviewer: ```suggestion
If this is an error, please send us an issue on our [Github Repository](https://github.com/c6o/docs/issues/new?assignees=&labels=bug&template=bug-report.md&title=) or by letting us know in [Discord](https://discord.gg/wx3JkVjTPy)
```
